src/colorize.cr

```diff
@@ -167,6 +167,51 @@ module Colorize
     def back(io : IO) : Nil
       (to_i + 10).to_s io
     end
+
+    def self.from_lscolors(color : String) : Color
```

Comment:
The two methods I added differ in their approach to error handling. The `?` method simply returns `nil` if it receives bad input. The other method throws an exception when it receives bad input. This pattern is seen in other places in the stdlib, for example `String#[]` and `String#[]?`.

I think you're right that `not_nil!` isn't the best option. Instead of a plain `Exception` with a message, it should throw something like a `LSColorNotFoundException`. What do you think?

Reply:
I've removed the exception-throwing method, leaving only the nilable version. Thanks for the feedback! 😄